### PR TITLE
Added whole-body manipulation notebook

### PR DESCRIPTION
**Pull request by cmastalli**
This PR includes a new notebook prepared for the summer MEMMO school.

@nmansard your feedback might be useful, but unfortunately, we need to merge it very soon.

@nim65s do we have time to merge this branch, and then to deploy a new Crocoddyl version?

### CODE REVIEW

File: examples/notebooks/whole_body_manipulation.ipynb [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+    "import example_robot_data\n",
+    "import numpy as np\n",
+    "\n",
+    "# Load robot\n",
+    "robot = example_robot_data.loadTalos()\n",
+    "rmodel = robot.model\n",
+    "q0 = rmodel.referenceConfigurations[\"half_sitting\"]\n",
+    "x0 = np.concatenate([q0, np.zeros(rmodel.nv)])\n",
+    "\n",
+    "# Declaring the foot and han names\n",
```
Comment (nim65s) [inline examples/notebooks/whole_body_manipulation.ipynb:53]: typo

File: examples/notebooks/whole_body_manipulation.ipynb [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+    "Let's solve this problem!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print \"Problem solved:\", fddp.solve()\n",
```
Comment (nim65s) [inline examples/notebooks/whole_body_manipulation.ipynb:236]: syntax error in python 3

File: examples/notebooks/whole_body_manipulation.ipynb [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the FDDP solver\n",
+    "fddp = crocoddyl.SolverFDDP(problem)\n",
+    "fddp.setCallbacks([crocoddyl.CallbackVerbose(), crocoddyl.CallbackDisplay(display)])\n",
+    "\n",
+    "# Solves the problem\n",
+    "print \"Problem solved:\", fddp.solve()\n",
```
Comment (nim65s) [inline examples/notebooks/whole_body_manipulation.ipynb:334]: syntax error

File: examples/notebooks/whole_body_manipulation.ipynb (hunk 1 above)
Comment (cmastalli) [reply examples/notebooks/whole_body_manipulation.ipynb]: done

File: examples/notebooks/whole_body_manipulation.ipynb (hunk 3 above)
Comment (cmastalli) [reply examples/notebooks/whole_body_manipulation.ipynb]: Fixed Python 3 compatibility

File: examples/notebooks/whole_body_manipulation.ipynb (hunk 2 above)
Comment (cmastalli) [reply examples/notebooks/whole_body_manipulation.ipynb]: same here

**Pull request merged**